src/test/groovy/transform/stc/GenericsSTCTest.groovy

```diff
@@ -1613,6 +1613,8 @@ assert result == 'ok'
             class ComponentFixture<T extends Component> {}
             class JButtonFixture extends ComponentFixture<JButton> {}
             class ContainerFixture<T extends Container> extends ComponentFixture<T> {}
+
+            @groovy.transform.TypeChecked(groovy.transform.TypeCheckingMode.SKIP)
```

Comment:
Yes, I thought it may have been the case that that class should not have compiled with STC in the first place. After you commented, I took another look and realized that it was an actual regression - pseudo property calls within the owning class would break with STC. I've updated my pull request accordingly. 

I also created a simpler unit test for pseudo property calls with STC, but that's probably better off in a separate pr/commit.